projects/canopy/src/lib/docs/principles/accessibility/developer.stories.mdx

```diff
@@ -0,0 +1,222 @@
+import { Meta } from '@storybook/addon-docs';
+import Feedback from '../../common/feedback.md';
+
+<Meta title="Principles/Accessibility/Developer" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# For developers
+
+<p class="standfirst">Here we provide information on developing content to be accessible.</p>
+
+## Developing with accessibility in mind
+
+A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times.
```

Comment:
Just a suggestion:

A great deal of web content can be made accessible by always using HTML elements for their correct purpose.

You can use WAI-ARIA, the Accessible Rich Internet Applications Suite, to make web content and web applications more accessible to people with disabilities. 

WAI-ARIA is especially helpful when creating dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.

Reply:
I've updated some of it, looks like this now:

```
A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times.

WAI-ARIA, the Accessible Rich Internet Applications Suite, can be used to make web content and web applications more accessible to people with disabilities.

WAI-ARIA is especially helpful when creating dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.
```